# 组内资料分享

## Paper-Reading-Group

List shared papers in our group
# 组内资料分享

## Paper-Reading-Group

List shared papers in our group

| Date      | Speaker | Paper                                                        | Slide                                                       | Keywords |
| --------- | :-----: | ------------------------------------------------------------ | ------------------------------------------------------------ | -------- |
| 2023.12.15 | 刘佳淑  | [《ACORDE unravels functionally interpretable networks of isoform co-usage from single cell data》(Nature Communications 2022)](https://doi.org/10.1038/s41467-022-29497-w) |[20221215_ljs.ppt](https://github.com/genemine/journalClub/blob/main/slides/20231215_ljs.pptx)| isoform共表达网络、单细胞 |
| 2023.11.08 | 段志伟  | [《MiXcan: a framework for cell-type-aware transcriptome-wide association studies with an application to breast cancer》(Nature communications 2023)](https://www.nature.com/articles/s41467-023-35888-4) |[mixcan.pptx](https://github.com/genemine/journalClub/blob/main/slides/mixcan.pptx)| TWAS |
| 2023.10.11 | 黄文魁  | [《Accurate proteome-wide missense variant effect prediction with AlphaMissense》(Science 2023)](https://doi.org/10.1126/science.adg7492) |[2023_AlphaMissense.ppt](https://github.com/genemine/journalClub/blob/main/slides/2023_AlphaMissense.pptx)| 错义突变致病性预测 |
| 2023.9.20 | 陈泽瑶  | [《A gene-based association method for mapping traits using reference transcriptome data》(Nature genetics 2015)](https://doi.org/10.1038/ng.3367) |[PrediXcan.pptx](https://github.com/genemine/journalClub/blob/main/slides/PrediXcan.pptx)| 单细胞 |
| 2022.9.20 | 黄文魁  | [《Variant to function mapping at single-cell resolution through network propagation》(Nature Biotechnology 2022)](https://doi.org/10.1038/s41587-022-01341-y) |[2022_SCAVENGE.ppt](https://github.com/genemine/journalClub/blob/main/slides/2022_SCAVENGE.pptx)| 单细胞 |
| 2022.9.06 | 刘佳淑  | [《Genetic control of RNA splicing and its distinct role in complex trait variation》(Nature Genetics 2022)](https://doi.org/10.1038/s41588-022-01154-4) |[THISTLE.ppt](https://github.com/genemine/journalClub/blob/main/slides/THISTLE.pptx)| sQTL   |
| 2022.06.16 | 张孝琪 | [《Graph Self-Supervised Learning: A Survey》](https://ieeexplore.ieee.org/abstract/document/9770382)| [图自监督学习.pptx](https://github.com/genemine/journalClub/blob/main/slides/%E5%9B%BE%E8%87%AA%E7%9B%91%E7%9D%A3%E5%AD%A6%E4%B9%A0.pptx)| 自监督学习 |
| 2022.03.02 | 张孝琪 | [《SEMI-SUPERVISED CLASSIFICATION WITH GRAPH CONVOLUTIONAL NETWORKS》(ICLR 2017)](https://openreview.net/forum?id=SJU4ayYgl)| [semiGCN.pptx](https://github.com/genemine/journalClub/blob/main/slides/semiGCN.pptx)| GCN |
| 2021.12.24 | 张超 | [《Ensemble dimensionality reduction and feature gene extraction for single-cell RNA-seq data》](https://www.nature.com/articles/s41467-020-19465-7)| [EDGE.pptx](https://github.com/genemine/journalClub/blob/main/slides/EDGE.pptx)| 单细胞降维和特征选择 |
| 2021.12.17 | 伍振鹏 | [《Deep neural networks identify sequence context features predictive of transcription factor binding》(*Nature Machine Intelligence 2021*)](https://www.nature.com/articles/s42256-020-00282-y)| [zheng2021AgentBind.pptx](https://github.com/genemine/journalClub/blob/main/slides/zheng2021AgentBind.pptx)| 转录因子结合预测 |
| 2021.11.11 | 刘海 | [《Prioritization of disease genes from GWAS using ensemble based positive unlabeled learning》(*nature  european journal of human genetics 2021*)](https://www.nature.com/articles/s41431-021-00930-w)| [PULearning.pptx](https://github.com/genemine/journalClub/blob/main/slides/PULearning.pptx)| 疾病基因预测 |
| 2021.12.03 | 刘坤 | [《GRiNCH: simultaneous smoothing and detection of topological units of genome organization from sparse chromatin contact count matrices with matrix factorization》(*Genome Biology 2021*)](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02378-z)| [20211203.pptx](https://github.com/genemine/journalClub/blob/main/slides/20211203.pptx)| TAD检测 |
| 2021.11.19 | 张硕 | [《MOGONET integrates multi-omics data using graph convolutional networks allowing patient classification and biomarker identification》(*Nat Commun 2021*)](https://www.nature.com/articles/s41467-021-23774-w)| [MOGONET.ppt](https://github.com/genemine/journalClub/blob/main/slides/MOGONET.pptx)| 多组学分类 |
| 2021.11.5 | 徐云佩  | [《Accurate feature selection improves single-cell RNA-seq cell clustering》(*Briefings in Bioinformatics. 2021*)](https://academic.oup.com/bib/article/22/5/bbab034/6145899?login=true)  |[FEAST.ppt](https://github.com/genemine/journalClub/blob/main/slides/FEAST.pptx)   |单细胞特征选择|
| 2021.7.6 | 张超  | [《Deep learning enables accurate clustering with batch effect removal in single-cell RNA-seq analysis》(*Nature communication. 2020*)](https://www.nature.com/articles/s41467-020-15851-3)  |[DESC.ppt](https://github.com/genemine/journalClub/blob/main/slides/DESC.pptx)   |单细胞聚类|
| 2021.6.25 | 刘祎玮  | [《Pathway-based subnetworks enable cross-disease biomarker discovery》(*Nature communication. 2018*)](https://www.nature.com/articles/s41467-018-07021-3)  |[SIMMS.ppt](https://github.com/genemine/journalClub/blob/main/slides/SIMMS.pptx)   |Pathway subnetwork及生存分析|
| 2021.6.23 | 伍振鹏  | [《Prevalent intron retention fine-tunes gene expression and contributes to cellular senescence》(*Aging Cell. 2020*)](https://onlinelibrary.wiley.com/doi/full/10.1111/acel.13276)  |[IR_senescence.ppt](https://github.com/genemine/journalClub/blob/main/slides/IR_senescence.pptx)   |内含子保留|
| 2021.6.1 | 刘坤  | [《SRHiC: A Deep Learning Model to Enhance the Resolution of Hi-C Data》(*Front. Genet. 2020*)](https://www.frontiersin.org/articles/10.3389/fgene.2020.00353/full)  |[SRHiC.ppt](https://github.com/genemine/journalClub/blob/main/slides/SRHiC.pptx)   |Hi-C数据增强|
| 2021.5.11 | 杨昌获  | [《PeNGaRoo, a combined gradient boosting and ensemble learning framework for predicting non-classical secreted proteins》(*Bioinformatics 2020*)](https://academic.oup.com/bioinformatics/article/36/3/704/5545087)  |[modelensemble.ppt](https://github.com/genemine/journalClub/blob/main/slides/modelensemble.pptx)   |模型集成|
| 2021.4.27 | 杨昌获  | [《PeNGaRoo, a combined gradient boosting and ensemble learning framework for predicting non-classical secreted proteins》(*Bioinformatics 2020*)](https://academic.oup.com/bioinformatics/article/36/3/704/5545087)  |[modelensemble.ppt](https://github.com/genemine/journalClub/blob/main/slides/modelensemble.pptx)   |模型集成|
| 2021.4.13 |  张硕   | [《INDUCTIVE MATRIX COMPLETION BASED ON GRAPH NEURAL NETWORKS》(*ICLR 2020*)](https://openreview.net/forum?id=ByxxgCEYDS)|[IGMC.pptx](https://github.com/genemine/journalClub/blob/main/slides/IGMC.pptx)|推荐系统|
| 2021.4.06 | 郑剑涛  |  [《Freddie: Annotation-independent Detection and Discovery of Transcriptomic Alternative Splicing Isoforms》(*bioRxiv 2021*)](https://www.biorxiv.org/content/10.1101/2021.01.20.427493v1.abstract)   |[Freddie.ppt](https://github.com/genemine/journalClub/blob/main/slides/Freddie.pptx)   | 三代测序；isoform预测|
| 2021.3.23 | 徐云佩  |  [《Deciphering cell–cell interactions and communication from gene expression》(*Nature Reviews Genetics 2020*)](https://www.nature.com/articles/s41576-020-00292-x) | [cell communication review.ppt](https://github.com/genemine/journalClub/blob/main/slides/cell%20communication%20review.pptx)  | 单细胞通讯 |
| 2021.3.18 | 方钊玉  | [《An entropy-based metric for assessing the purity of single cell populations》(*Nat Commun 2020*)](https://www.nature.com/articles/s41467-020-16904-3) |[ROGUE.ppt](https://github.com/genemine/journalClub/blob/main/slides/ROGUE.pptx)| 单细胞   |
